battleship.py

```diff
@@ -185,7 +207,10 @@ def drawShip(data, canvas, ship):
 Returns: bool
 '''
 def shipIsValid(grid, ship):
-    return
+    if checkShip(grid,ship):
+      if isHorizontal(ship) or isVertical(ship):
+        return True
+    return False

```

Comment:
Check the length of the ship also

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py